docs/source/api/v2/federations_all.rst

```diff
@@ -0,0 +1,92 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-federations-all:
+
+*******************
+``federations/all``
+*******************
+
+``GET``
+=======
+Retrieves a list of :term:`Federation` mappings (also called :term:`Federation` Resolvers) for the current user.
+
+:Auth. Required: Yes
+:Roles Required: "admin"
+:Response Type:  Array
+
+Request Structure
+-----------------
+No parameters available.
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/2.0/federations/all HTTP/1.1
+	User-Agent: python-requests/2.22.0
+	Accept-Encoding: gzip, deflate
+	Accept: */*
+	Connection: keep-alive
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+:deliveryService:       The :ref:`ds-xmlid` of the delivery service.
+:mappings:              An array of objects that represent the mapping of a :term:`Federation`'s :abbr:`CNAME (Canonical Name)` to one or more Resolvers
+	:cname:                 The actual CNAME used by the :term:`Federation`
```

Comment:
You need a blank line between a list item and the beginning of a sub list in order for it to not parse like a definition list item.

Reply:
Blank line added in de689b9e6. Looking at the output of

```shell
pcregrep -rMn '^:.*\n'$'\t'':' .
```

that was the only occurrence.